infra/taskcluster-hook-test-select.json

```diff
@@ -74,6 +76,10 @@
     "triggerSchema": {
         "additionalProperties": false,
         "properties": {
+            "PHABRICATOR_DEPLOYMENT": {
+                "type": "string",
+                "enum": ["prod", "dev"]
```

Comment:
Why not stick to production/testing and use `APP_CHANNEL` on the code-review side ?

Reply:
Having a testing channel that does exactly the same things as production would be quite expensive, so I only have a production channel for bugbug

Reply:
Ah you meant passing the APP_CHANNEL from code-review. That's a problem because of https://github.com/mozilla/code-review/pull/433#discussion_r365708075